features/step_definitions/i14y_drawer_steps.rb

```diff
@@ -3,11 +3,11 @@
  # See https://github.com/GSA/usasearch/blob/master/README_I14Y.markdown
 
 Given /^we don't want observers to run during these cucumber scenarios$/ do
-  ActiveRecord::Base.observers.disable :all
```

Comment:
This and subsequent calls to `ActiveRecord::Base.observers` should be left as `ActiveRecord::Base.observers`. The observer classes still inherit from `ActiveRecord::Observer`, and can be called using `ActiveRecord::Base.observers`. See https://github.com/rails/rails-observers.
```ruby
> ActiveRecord::Base.observers
[
    [0] :sayt_filter_observer,
    [1] :misspelling_observer,
    [2] :indexed_document_observer,
    [3] :affiliate_observer,
    [4] :navigable_observer,
    [5] :searchable_observer,
    [6] :rss_feed_url_observer,
    [7] :i14y_drawer_observer,
    [8] :routed_query_keyword_observer,
    [9] :watcher_observer
]
```

Reply:
ApplicationRecord inherits from ActiveRecord::Base so this is fine.  And if you remember https://github.com/GSA/search-gov/pull/329 tests were failing till I changed it to ApplicationRecord.  
See
https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-2-to-rails-5-0